index.js

```diff
@@ -70,7 +70,7 @@ function enableModuleReplacement(opts) {
     watching[path] = watch(path, { persistent: false }, function(eventType, filename) {
       const oldModule = require.cache[path];
 
-      const deps = collectDependencies(oldModule);
+      const deps = oldModule ? collectDependencies(oldModule) : [];
```

Comment:
In my tracing, `require.cache[.../.next/build-manifest.json]` doesn't exist in the cache anymore (for whatever reason beyond our control), so this simply says there are no dependencies for it then.